modules/management/kubectl-apply/variables.tf

```diff
@@ -92,7 +95,7 @@ variable "kueue" {
   description = "Install and configure [Kueue](https://kueue.sigs.k8s.io/docs/overview/) workload scheduler. A configuration yaml/template file can be provided with config_path to be applied right after kueue installation. If a template file provided, its variables can be set to config_template_vars."
   type = object({
     install              = optional(bool, false)
-    version              = optional(string, "v0.12.2")
+    version              = optional(string, "0.13.2")
```

Comment:
Why is the default version not 0.14.0 if that is also being supported as per line 21?